pages/about.html

```diff
@@ -5,8 +5,25 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <header>
+    <h5>Michelle's portfolio website</h5>
```

Comment:
I recommend using "h" tags for structure rather than styling. As this is the text of the header, I'd recommend using 'h1' here and then styling it with CSS.